docs/embedded/administration/developer-admin/dev-admin.md

```diff
@@ -54,7 +55,7 @@ To get started using PowerShell to manage SharePoint Embedded, you have to insta
 
 ### Creation of container types
 
-The developer administrator can create a container type using PowerShell cmdlets. Each container type is associated to an application ID, a one to one mapping, and an Azure subscription ID. The developer administrator can also create Trial container types that have a validity of 30 days to test out SharePoint Embedded. The following commands can be used to create SharePoint Embedded container types on the developer admin’s tenant:
+The developer administrator can create a container type using PowerShell cmdlets. Each container type is associated to an application ID, a one to one mapping, and an Azure subscription ID. The developer administrator can also create Trial container types that have a validity of 30 days to test out SharePoint Embedded. The following [commands](https://learn.microsoft.com/en-us/powershell/module/sharepoint-online/new-spocontainertype?view=sharepoint-ps) can be used to create SharePoint Embedded container types on the developer admin’s tenant:
```

Comment:
Per the automated build report, Invalid link - please fix:

- don't use absolute links, use relative links
- don't include locale in link
- remove the preview link from the URL - target page only has powershell code listings, so no need to set the view